apps/src/storage/dataBrowser/DataWorkspace.jsx

```diff
@@ -1,4 +1,4 @@
-import { ApplabInterfaceMode } from '../constants';
+import { ApplabInterfaceMode } from '../../applab/constants';
```

Comment:
Are you planning to use the data browser in gamelab? This looks like the only remaining dependency from `storage` to `applab`. You don't need to break this dependency now, but you probably should before you start using this in gamelab.